### PR TITLE
add highchair field to cafe,fast_food,restaurant

### PR DESCRIPTION
**Pull request by andrewharvey**
The wiki https://wiki.openstreetmap.org/wiki/Key:highchair says it can be yes/no or the number of chairs available. It doesn't look like these is support here yet for a yes/no/number and given yes/no is more important than the number I've set it to be yes/no only.

### CODE REVIEW

File: data/fields/highchair.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "key": "highchair",
+    "type": "check",
+    "label": "Highchair",
+    "autoSuggestions" : false,
+    "customValues" : true,
```
Comment (tyrasd) [inline data/fields/highchair.json:6]: for `type=check` fields, these settings don't do anything, or am I overlooking something?

**Review by tyrasd — COMMENTED** [review]
thanks! this looks generally quite good :+1: but please note the two comments below.

File: data/fields/highchair.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "key": "highchair",
+    "type": "check",
+    "label": "Highchair",
```
[the remark this answers is not in the document]
Comment (andrewharvey) [reply data/fields/highchair.json]: shouldn't the file name match the non-separated key?

[the remark this answers is not in the document]
Comment (tyrasd) [reply data/fields/highchair.json]: You are [right](https://github.com/ideditor/schema-builder#preset-files). Sorry for the confusion.

**Pull request merged**